crates/builder/sequencer.openapi.yml

```diff
@@ -0,0 +1,143 @@
+              required:
+              - host_block_number
+              - host_chain_id
+              - ru_chain_id
+              - gas_limit
+              - ru_reward_address
+              - contents
+      responses:
+        '200':
+          description: A JSON SignResponse
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  req:
+                    type: object
+                    description: The inputted SignRequest.
+                    properties:
+                      host_block_number:
+                        type: string
+                        description: The block number of the host.
+                      host_chain_id:
+                        type: string
+                        description: The chain ID of the host.
+                      ru_chain_id:
+                        type: string
+                        description: The chain ID of the rollup.
+                      gas_limit:
+                        type: string
+                        description: The gas limit of the rollup block.
+                      ru_reward_address:
+                        type: string
+                        description: The reward address on the rollup for the builder.
+                      contents:
+                        type: string
+                        description: keccak256 hash of rlp-encoded transactions in the block.
+                  sig:
+                    type: object
+                    description: The signature over the SignRequest.
+                    properties:
+                      v:
+                        type: string
+                        description: Signature V field.
+                      r:
+                        type: string
+                        description: Signature R field.
+                      s:
+                        type: string
+                        description: Signature S field.
+components:
+  schemas:
+    SignRequest:
+      type: object
+      properties:
+        host_block_number:
+          type: string
+          description: The block number of the host.
+        host_chain_id:
+          type: string
+          description: The chain ID of the host.
+        ru_chain_id:
+          type: string
+          description: The chain ID of the rollup.
+        gas_limit:
+          type: string
+          description: The gas limit of the rollup block.
+        ru_reward_address:
+          type: string
+          description: The reward address on the rollup for the builder.
+        contents:
+          type: string
+          description: keccak256 hash of rlp-encoded transactions in the block.
+      required:
+      - host_block_number
+      - host_chain_id
+      - ru_chain_id
+      - gas_limit
+      - ru_reward_address
+      - contents
+    Signature:
+      type: object
+      properties:
+        v:
```

Comment:
i believe this is actually incorrect. The V value is serialized based on parity rules. This _should_ be a boolean called `yParity`, and if we want to guarantee that, we can do so in quincey